modular_skyrat/modules/synths/code/species/synth_prefab.dm

```diff
@@ -0,0 +1,19 @@
+/mob/living/carbon/human/species/synth
+	race = /datum/species/synthetic
+
+// Used for printing dead synth bodies
+/mob/living/carbon/human/species/synth/empty
+
+/mob/living/carbon/human/species/synth/empty/Initialize(mapload)
+	. = ..()
+	var/old_death_sound = death_sound // You can't do this with initial() as the death sound is set after by the species datum
```

Comment:
Technically, they don't have a deathsound, but this is still futureproofed just in case